### PR TITLE
Docs: ICP <> BTC and ICP <> ETH developer workflows

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Each of these steps can be executed programmatically within a smart contract that is deployed on ICP. View the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation to see code examples for each RPC method.
+
+## Integrating with Ethereum
+
+The Ethereum integration can be incorporated into dapps for several purposes, such as authentication, as shown in the [Sign In With Ethereum](siwe.mdx) example. Additional sample projects can be found in the [EVM RPC](../evm-rpc/samples.mdx) canister documentation.
+
+## Using ckETH
+
+Chain-key Ethereum (ckETH) is a digital twin token of Ethereum that is backed 1:1 with Ethereum held within an ICP smart contract. ckETH tokens comply with the [ICRC token standard](/docs/current/developer-docs/multi-chain/ethereum/cketh/overview).
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:46]: Should we also mention ckUSDCSepolia as well?

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    - `eth_feeHistory`: Returns the historical fee data to estimate gas prices for transactions.
+
+    - `eth_getLog`s: Returns the logs of a specified block or transaction.
+
+Each of these steps can be executed programmatically within a smart contract that is deployed on ICP. View the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation to see code examples for each RPC method.
+
+## Integrating with Ethereum
+
+The Ethereum integration can be incorporated into dapps for several purposes, such as authentication, as shown in the [Sign In With Ethereum](siwe.mdx) example. Additional sample projects can be found in the [EVM RPC](../evm-rpc/samples.mdx) canister documentation.
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:42]: I suggest moving the example of EVM RPC to Step 4 above as it relates to the EVM RPC.

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    - `eth_getBlockByNumber`: Returns information about a given block.
+
+    - `eth_getTransactionCount`: Returns the number of transactions for a specified address.
+
+    - `eth_getTransactionReceipt`: Returns details about a submitted transaction.
+
+    - `eth_feeHistory`: Returns the historical fee data to estimate gas prices for transactions.
+
+    - `eth_getLog`s: Returns the logs of a specified block or transaction.
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:36]:
```suggestion
    - `eth_getLogs`: Returns the logs of a specified block or transaction.
```

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Each of these steps can be executed programmatically within a smart contract that is deployed on ICP. View the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation to see code examples for each RPC method.
+
+## Integrating with Ethereum
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:40]: If we remove the EVM RPC note, maybe we can make this section about SIWE wallet authentication only?

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx (hunk 1 above)
Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:46]: Added info about ckUSDCSepolia

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx (hunk 2 above)
Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:42]: Revised per feedback

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx (hunk 4 above)
Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:42]: Removed EVM RPC info and added additional SIWE info

File: docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Chain-key Ethereum (ckETH) is a digital twin token of Ethereum that is backed 1:1 with Ethereum held within an ICP smart contract. ckETH tokens comply with the [ICRC token standard](/docs/current/developer-docs/multi-chain/ethereum/cketh/overview).
+
+ckETH and other chain-key tokens can be interacted with in the same manner as other ICRC tokens. There aren't any special workflow steps, parameters, or smart contract types that must be used to integrate with chain-key tokens such as ckETH.
+
+[Learn more about ckETH](/docs/current/developer-docs/multi-chain/ethereum/cketh/overview).
+
+## Using ckERC20 tokens
+
+Current, the ckERC20 token ckUSDCSepolia is available for testing. This token is still under development and there may be bugs.
```
Comment (jennifertrin) [inline docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx:56]:
```suggestion
Currently, the ckERC20 token ckUSDCSepolia is available for testing. This token is still under development and there may be bugs. 
```

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**